tables/tr-g1.ctb

```diff
@@ -30,19 +30,179 @@
 
 # Used to display symbols, Unicode chars and other symbols used
 # throughout UEB code.
+lowercase q 4-12345
+lowercase w 4-2456
+lowercase x 4-1346
+# Turkish characters (added by Arend Arends) and Oguz Ogur
 
-include en-ueb-g1.ctb
```

Comment:
Why did you remove this include? Was it so that you can overwrite the basic latin capital i? Could you not keep the include and overwrite the definition of the i in some other way?

Reply:
Hi Chris,

Yes, exactly, the different i letters related rule needs I need moved the include line from top of tr-g1.ctb table to end of tr-g1.ctb table.
When I experimented with entering different variations of the letter i so that the include line was at the beginning of the table in the original place, the correct order of precedence simply did not take effect, the point changes requested by Oguz did not take effect. However, according to the read grade1 test result, all characters are good in the case of the tr-g1.ctb table, this was also confirmed by Oguz.
Of course this is true only when the include line is have the end of the tr-g1.ctb table.
If I put a hassmark the include en-ueb-g1.ctb line, the tr-g1.yaml produce 155 failure results.
If I putting back the 38TH line the include en-ueb-g1.ctb line, the grade1 turkish test file produce I think 168 failures.
So, I am fraid this is the best combination (keeping the end of table the include en-ueb-g1.ctb include line).
This way when I requested Oguz the grade 1 test entire proofread, he says all dot combination is right if we use this variation.

Reply:
OK, sounds good